### PR TITLE
changes needed to build using Elm 0.18

### PR DESCRIPTION
**Pull request by dc25**
With these changes the elm-pong project should build under Elm 0.18 .  

I tried to preserve the style of the original code by keeping changes minimal.

### CODE REVIEW

**Comment by davydog187** [comment]
Whoa! Thanks for the upgrade!

File: elm-package.json [hunk 1]
```diff
@@ -1,15 +1,19 @@
 {
     "version": "1.0.0",
-    "summary": "Game of pong, based on Evan's blog post. Enhanced with some new features",
-    "repository": "https://github.com/davydog187/elm-pong.git",
+    "summary": "helpful summary of your project, less than 80 characters",
+    "repository": "https://github.com/user/project.git",
```
Comment (davydog187) [inline elm-package.json]: Can you revert these two lines?

**Comment by davydog187** [comment]
Feel free to add yourself as an author! I really appreciate you putting in these fixes!

File: elm-package.json (hunk 1 above)
Comment (dc25) [inline elm-package.json]: Whoops.  I fixed it.  Was generating from scratch with each new elm version and forgot that there was hand written information in there.

**Comment by dc25** [comment]
You're welcome.  Thanks for merging the changes.   This stackoverflow question : http://stackoverflow.com/questions/41408360/what-happened-to-graphics-package-in-elm-0-18 is what got me started down the path of updating your project.

**Comment by davydog187** [comment]
News to me, thanks again @dc25 !

**Pull request merged**